ga4gh/exceptions.py

```diff
@@ -0,0 +1,85 @@
+"""
+Module containing error objects we return to clients
+"""
+
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import ga4gh.protocol as protocol
+
+
+class ParentException(Exception):
```

Comment:
I'd probably rather call this `ServerError`, but let's see what the outcome of the Exception vs Error discussion is.